tensorflow/python/eager/def_function.py

```diff
@@ -557,6 +557,12 @@ def _get_tracing_count(self):
     result += self._stateful_fn.tracing_count if self._stateful_fn else 0
     return result
 
+  def experimental_get_tracing_count(self):
```

Comment:
`_get_tracing_count(...)` is only used in this file.  Could you instead rename `_get_tracing_count` to `experimental_get_tracing_count` in this file? (with your function comment)

Reply:
fixed.